docs/installing.md

```diff
@@ -1,25 +1,50 @@
-# Running a local Haveno test network
+# Set up environment
 
-These are the steps needed to set up Haveno test instances.
+These are the steps needed to build Haveno and test it on our test network or locally.
 
-## 1. Install dependencies
+## Install dependencies
 
-On Ubuntu: `sudo apt install make wget git git-lfs openjdk-11-jdk`. The Bitcoin and Monero binaries will be downloaded and verified automatically in the next step.
+On Ubuntu: `sudo apt install make wget git git-lfs openjdk-11-jdk`. The Monero binaries will be downloaded and verified automatically in the next step.
 
-## 2. Build Haveno
+## Build Haveno
 
 1. Download this repository: `git clone https://github.com/haveno-dex/haveno.git`
-2. Navigate to the root of the repository (`cd haveno`) and build the repository: run `make` in the terminal and wait until the process is completed (this will also download and verify the Monero and Bitcoin binaries).
+2. Navigate to the root of the repository (`cd haveno`) and build the repository: run `make` in the terminal and wait until the process is completed (this will also download and verify the Monero binaries).
 
-## 3. Run a local XMR testnet
+## Join the public testnet
+
+If you want to try Haveno in a live setup, Launch an Haveno instance that will connect to other peers on our public test environment, which runs on Monero's stagenet. You'll be able to make test trades with other users and have a preview of Haveno's trade protocol in action. Note that development is very much ongoing. Things are slow and might break.
```

Comment:
launch should be lowercase.

maybe debatable, but I'd use "a Haveno" over "an Haveno"